cypress/integration/bonus.test.js

```diff
@@ -4,7 +4,7 @@ describe("Bonus", () => {
     cy.get("input").type("say hello");
     cy.get("form").submit();
     cy.get("#todos").contains("say hello");
-    cy.contains("New todo added: say hello");
+    cy.get(".Toastify__toast--success").contains("New todo added: say hello");
```

Comment:
This ups the challenge by requiring the toast to have the right `type` property. I'm fine with this (it is also what the gif in the README shows) but in this case we should probably make that requirement clear in the README. I.e., when the todo is created, the toast should be type `success`, when the todo is deleted, the toast should be type `error`, for the bonus use the default type (no type specified).

The alternative is to only look for some kind of base `.Toastify` class if that exists on the element

Reply:
Is this not clear enough?

> - Use `componentDidMount` in `Todo` to show a `"success"` toast with the text `New todo added: {todo}`
> - Use `componentWillUnmount` in `Todo` to show an `"error"` toast with the text `Todo deleted: {todo}`

I'll defer to you on phrasing, whatever you think is appropriate 😄

Reply:
Ah, I think I missed that. Thank you!